modules/ebi-metagenomics/librarystrategycheck/meta.yml

```diff
@@ -0,0 +1,47 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "librarystrategycheck"
+description: Script that makes a good guess about whether reads are AMPLICON or NOT AMPLICON
+keywords:
+  - sequencing
+  - reads
+  - qc
+  - amplicon
+tools:
+  - "librarystrategycheck":
+      description: "Script that makes a good guess about whether reads are AMPLICON or NOT AMPLICON, uses mgnify-pipelines-toolkit as conda environment"
+      homepage: "https://github.com/EBI-Metagenomics/mgnify-pipelines-toolkit"
+      documentation: "https://github.com/EBI-Metagenomics/mgnify-pipelines-toolkit"
+      tool_dev_url: "https://github.com/EBI-Metagenomics/mgnify-pipelines-toolkit"
+      licence: ["Apache-2.0"]
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'sample1', single_end:false ]`
+  - mcp_cons:
+      type: file
+      description: MCP conservation curve file output by `assess_mcp_proportions` tool, also from the mgnify-pipelines-toolkit
+      pattern: "*.tsv"
+
+output:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'sample1', single_end:false ]`
+  - txt:
+      type: file
+      description: Text file containing the string "AMPLICON" if classified as AMPLICON, empty file otherwise
```

Comment:
We should use env(is_amplicon) instead of a file, maybe we can store the file anyway (to make it easier to debug). Like https://github.com/EBI-Metagenomics/mettannotator/blob/main/modules/local/lookup_kingdom.nf#L13